src/sentry/logging/formatters.py

```diff
@@ -0,0 +1,33 @@
+"""
+sentry.logging.formatters
+~~~~~~~~~~~~~~~~~~~~~~~~~
+:copyright: (c) 2010-2016 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+
+from __future__ import absolute_import
+
+import traceback
+
+from logging import Formatter
+
+import msgpack
+
+from django.utils.encoding import force_bytes
+
+
+class MessagePackFormatter(Formatter):
+    """
+    Return a packed dictionary.
+
+    Will pack the dictionary or the String representation of the message.
+    """
+    def format(self, record):
+        pack = record.msg if isinstance(record.msg, dict) else {
+            'msg': force_bytes(record.msg, errors='replace')
+        }
+        pack['levelname'] = record.levelname
+        if record.exc_info:
+            pack['traceback'] = traceback.format_tb(record.exc_info[-1])
+
+        return msgpack.packb(pack)
```

Comment:
I'd do `from msgpack import packb`, then call just `packb` to be a tiny bit faster.